DirectProgramming/Fortran/EdgeDetection/sobel-edge-detection/sample.json

```diff
@@ -0,0 +1,24 @@
+{
+  "guid": "28C99D5B-0E28-4D2A-BE50-8154264F7C48",
+  "name": "Sobel Edge Detection",
+  "categories": ["Toolkit/oneAPI Direct Programming/Fortran/OpenMP"],
+  "description": "Offload Fortran loops to a GPU",
+  "toolchain": [ "ifx" ],
+  "languages": [ { "fortran": {} } ],
+  "targetDevice": [ "CPU", "GPU" ],
```

Comment:
@hagabb This should be ``"targetDevice": ["GPU"]``.

Code works on GPU but if you try to run it on CPU I think it will crash.

Reply:
Why do you think it will crash on the CPU? The makefile creates working executables for sequential CPU, parallel CPU, and GPU offload. The description just means that this example shows how to offload Fortran loops if that's what the user wants.

Reply:
@hagabb My mistake. I had an issue running the sample before but it's now resolved. I will approve it.